js/imageViewerAnnotate.js

```diff
@@ -185,9 +203,17 @@ IVEM.insertPreview = function (field, params) {
       $container = $("<div></div>")
         .attr("data-ivem-container", params.container_id)
         .css("position", "relative")
-        .css("padding-top", "30px")
-        .css("margin-left", "auto")
-        .css("margin-right", "auto");
+        .css("margin-bottom", "40px")
+        .css("height", "50vh")
+        .css("max-width", "300px");
```

Comment:
why the `max-width`? along with the `height` should we set a `min-height` and a `max-height` for the case of very small/large screens?

Reply:
the main issue I was having was finding the right `max-height` that didn't distort the images whereas setting a `max-width` seemed to autoscale the image without any distortion

Reply:
intriguing.  What's this look like with a landscape image on desktop?